code/_onclick/hud/screen_objects.dm

```diff
@@ -69,7 +69,7 @@
 /obj/screen/craft/Click()
 	if(isobserver(usr))
 		return
-	SEND_SIGNAL(usr, COMSIG_CRAFT_MENU_OPEN, usr)
+	SEND_SIGNAL(src, COMSIG_CLICK, usr)

```

Comment:
click is already sent on the base of the proc, you dont need any of this proc any more